reliability-engineering/terraform/modules/concourse-monitoring/files/dashboards/metrics-by-team.json

```diff
@@ -0,0 +1,692 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 1,
+  "id": 4,
+  "iteration": 1590154848648,
+  "links": [],
+  "panels": [
+    {
+      "content": "---\n\n✈️ [Concourse](https://cd.gds-reliability.engineering)\n\n🔥 [Prometheus 1](https://prom-1.monitoring.cd.gds-reliability.engineering/graph)\n\n🔥 [Prometheus 2](https://prom-2.monitoring.cd.gds-reliability.engineering/graph)\n\n---",
```

Comment:
since there is an intention in other PR to run prom in staging too ... these hard coded "monitoring.cd.gds" domains should probably be templated out ... or just dropped in this case since they are just infomational links

Reply:
I'm just recording the existing dashboards rather than trying to change them.
I probably already need to overwrite this with the latest version